packages/rest/src/validation/ajv-factory.provider.ts

```diff
@@ -29,12 +30,17 @@ const ajvErrors = require('ajv-errors');
  */
 @bind({scope: BindingScope.SINGLETON})
 export class AjvFactoryProvider implements Provider<AjvFactory> {
+  static readonly defaultValidationOptions: RequestBodyValidationOptions = {
+    $data: true,
+    ajvKeywords: true,
+    ajvErrors: true,
+  };
```

Comment:
IMO, a constant is probably simpler here:
```ts
export DEFAULT_AJV_VALIDATION_OPTIONS: RequestBodyValidationOptions = {
  $data: true,
  ajvKeywords: true,
  ajvErrors: true,
};
```

Reply:
Now the ajv validation is used beyond request body. We probably should rename `RequestBodyValidationOptions` to `ValidationOptions` or `AjvValidationOptions`. For backward compatibility:

```ts
export type RequestBodyValidationOptions = ValidationOptions;
```